src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPage.vb

```diff
@@ -340,13 +340,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             If Not _propPage Is Nothing Then
 
                 _propPage.SuspendLayout() 'No need for more layouts...
-                If _propPage.Parent IsNot Nothing AndAlso Not _hostedInNative Then
-                    'We sited ourselves by setting the Windows Forms Parent property
-                    _propPage.Parent = Nothing
-                ElseIf _wasSetParentCalled Then
-                    'We sited ourselves via a native SetParent call
-                    NativeMethods.SetParent(_propPage.Handle, _prevParent)
-                End If
+                NativeMethods.SetParent(_propPage.Handle, _prevParent)
```

Comment:
Haven't seen any (including for scaling in high-DPI scenarios), and we had the debug option to force this behavior already.